Assets/HoloToolkit/Input/Tests/KeywordManager.unity

```diff
@@ -0,0 +1,513 @@
+%YAML 1.1
```

Comment:
None of the other *.cs files in the Tests/Scripts folder have copyright headers. Should I add it to all of them?